src/Codeception/Command/Run.php

```diff
@@ -405,18 +408,60 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         // Run all tests of given suite or all suites
         if (!$test) {
-            $suites = $suite ? explode(',', $suite) : Configuration::suites();
-            $this->executed = $this->runSuites($suites, $this->options['skip']);
-
-            if (!empty($config['include']) and !$suite) {
+            $raw_suites = $suite ? explode(',', $suite) : Configuration::suites();
+            
+            /** @var string[] $main_app_suites */
+            $main_app_suites = [];
```

Comment:
Please rename all your variables to snakeCase, e.g. `$mainAppSuites`.

Reply:
@Naktibalda done